### PR TITLE
Update host is possible

### PR DESCRIPTION
**Pull request by elodie-nazaret**

### CODE REVIEW

File: src/Service/SiteService.php [hunk 1]
```diff
@@ -63,6 +64,24 @@ public function postSite(SitePostMessage $message): Site
     }
 
     /**
+     * @param SitePatchMessage $message
+     *
+     * @return Site
+     */
```
Comment (barberemi) [inline src/Service/SiteService.php]: You can delete doc maybe?

File: src/Message/Site/SitePatchMessage.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        return [
+            'host' => $this->getHost(),
+        ];
+    }
+
+    /**
+     * @param Site $site
+     *
+     * @return self
+     */
```
Comment (barberemi) [inline src/Message/Site/SitePatchMessage.php]: You can delete doc maybe

File: src/Service/SiteService.php (hunk 1 above)
Comment (tristanbes) [inline src/Service/SiteService.php]: I'd rather not because it's available to public.

File: src/Message/Site/SitePatchMessage.php (hunk 2 above)
Comment (tristanbes) [inline src/Message/Site/SitePatchMessage.php]: same

**Pull request merged**